src/lib/tablet-full-screen.js

```diff
@@ -0,0 +1,15 @@
+import bowser from 'bowser';
+
+/**
+ * Helper method to request full screen in the browser when on a tablet.
+ */
+export default function () {
+    if (bowser.tablet) {
+        if (bowser.webkit || bowser.blink) {
+            document.documentElement.webkitRequestFullScreen();
```

Comment:
I think we should be a bit more defensive with unstable vendor prefixed APIs like this, could you check that it exists first, like `document.documentElement.webkitRequestFullScreen && document.documentElement.webkitRequestFullScreen()`?

Reply:
👍